gui/default/syncthing/device/editDeviceModalView.html

```diff
@@ -137,11 +137,22 @@
               </div>
             </div>
           </div>
-          <div class="row form-group">
-            <div class="col-md-12">
+          <div class="row">
+            <div class="col-md-6" ng-class="{'has-error': deviceEditor.numConnections.$invalid && deviceEditor.numConnections.$dirty}">
+              <label translate>Connection Management</label>
+              <div class="row">
+                <span class="col-md-8" translate>Number of Connections</span>
+                <div class="col-md-4">
+                  <input name="numConnections" id="numConnections" class="form-control" type="number" pattern="\d+" ng-model="currentDevice.numConnections" min="1" />
+                </div>
+              </div>
+              <p class="help-block" ng-if="!deviceEditor.numConnections.$valid && deviceEditor.numConnections.$dirty" translate>The number of connections must be a positive number</p>
+              <p class="help-block" ng-if="deviceEditor.numConnections.$valid || deviceEditor.numConnections.$pristine">When set to more than one, Syncthing will attempt to establish multiple concurrent connections. One connection is reserved for control messages, the rest are used for data transfer.</p>
```

Comment:
What exactly are "control messages"? I've got a feeling the user won't be able to understand part without a longer explanation (e.g. a through a link to the Docs).

Reply:
I don't think it matters too much if the precise definition is understood as it's followed by "the rest are used for data transfer", but feel free to suggest something. As for when it's set to one, yes, obviously everything will run on that one connection like it's always done.

Reply:
I would just add this clarification, but when it comes to "control messages", I can't really suggest anything, as I'm also not sure what exactly they are 🙁.

```suggestion
              <p class="help-block" ng-if="deviceEditor.numConnections.$valid || deviceEditor.numConnections.$pristine">When set to more than one, Syncthing will attempt to establish multiple concurrent connections. One connection is reserved for control messages, the rest are used for data transfer. When set to one, the single connection is used for both.</p>
```